### ISSUE TITLE
Stale bot spam

### ISSUE THREAD

**Issue by Darthagnon** [g1eny0ung/hugo-theme-dream#277]
We're developing a small website theme as a hobby in our limited free time, not the Chromium source code here. The stale bot spamming and closing issues is counterproductive and quite annoying.  [My pull request](https://github.com/g1eny0ung/hugo-theme-dream/pull/247) has also been closed by this robot. Can we please disable it for this repo?

I am still actively working on improvements to this theme in [my repo](https://github.com/Darthagnon/hugo-theme-dream/tree/production), more or less as a hard fork. I haven't updated my pull request in a while as I haven't had time to split the branches out again after merging them. Most of my changes are cumulative, anyway, but that style of development doesn't appear to suit contribution guidelines for this repo.

**Comment by g1eny0ung** [comment]
@Darthagnon Yep. I also realize this problem. I've already deleted this stable bot in this repo. Thanks for reminding me to handle this problem.

> Most of my changes are cumulative, anyway, but that style of development doesn't appear to suit contribution guidelines for this repo.

About the PR https://github.com/g1eny0ung/hugo-theme-dream/pull/247, and this one, to be honest, there's something too "cumulative" about the changes here. There are so many changes here that I'm really having a hard time dealing with up to 200+ changes. 🤯

I noticed your comments on https://github.com/g1eny0ung/hugo-theme-dream/pull/247#issuecomment-1152811351, it's making it difficult for me. The issue at hand is that if I merge this PR, I may miss out on several critical details of the master branch down the line. It seems like there wasn't a comprehensive summary of the specifics provided in this PR. In my experience with open source, this inevitably creates more problems later on, because if something goes wrong somewhere, I need to recall it from the very beginning of the context (yes, this PR), which will take more time.

I want to make it clear that I am not blaming or complaining, but rather, I am willing to embrace the change and move forward. Shall we begin with the modifications in the docs folder? I will extract the alterations, review them, and once they are merged, then merge the master branch into this pull request. This will separate that part of the changes from the PR.
